src/validate.ts

```diff
@@ -12,18 +12,15 @@ const DISCONNECT_RESPONSE_SCHEMA = require('../intents/disconnect.response.schem
 const ON_OFF_STATES_SCHEMA = require('../traits/onoff.states.schema.json');
 const ON_OFF_ATTRIBUTES_SCHEMA = require('../traits/onoff.attributes.schema.json');
 
+
 const COMMAND_STATES_EXPECT = {
-  'action.devices.commands.OnOff': ON_OFF_STATES_SCHEMA,
+  'action.devices.traits.OnOff': ON_OFF_STATES_SCHEMA,
```

Comment:
This change would affect 'action.devices.commands.OnOff' and break the Execute validation below.

Reply:
Changes made to have two separate query and execute dictionaries